### PR TITLE
koodo-reader 1.7.2

### PR DESCRIPTION
**Pull request by eleanordoesntcode**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.
---

### CODE REVIEW

File: Casks/k/koodo-reader.rb [hunk 1]
```diff
@@ -1,11 +1,12 @@
 cask "koodo-reader" do
-  arch arm: "-arm64"
+  arch arm: "arm64",
```
Comment (bevanjkay) [inline Casks/k/koodo-reader.rb:2]:
```suggestion
  arch arm: "arm64"

**Comment by eleanordoesntcode** [comment]
I thought I had removed that dash. Weird.

File: Casks/k/koodo-reader.rb [hunk 2]
```diff
@@ -1,11 +1,12 @@
 cask "koodo-reader" do
-  arch arm: "-arm64"
+  arch arm: "arm64"
+       intel: "x64"
```
Comment (bevanjkay) [inline Casks/k/koodo-reader.rb:3]:
```suggestion
  arch arm: "arm64", intel: "x64"
```

Sorry, I read this wrong the last time around.

Comment (eleanordoesntcode) [reply Casks/k/koodo-reader.rb]: ah I already did the modification on my end sorry

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**